docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -745,9 +745,13 @@ https://docs.microsoft.com/en-us/sql/t-sql/functions/format-transact-sql?view=sq
 If any of the two arguments is `null` or the pattern is an empty string `null` is returned.
 
 [NOTE]
-[1] If the 1st argument is of type `time`, then pattern specified by the 2nd argument cannot contain date related units
-(e.g. 'dd', 'MM', 'YYYY', etc.). If it contains such units an error is returned. +
-[2] Format specifier `F` will be working similar to format specifier `f`.
+If the 1st argument is of type `time`, then pattern specified by the 2nd argument cannot contain date related units
+(e.g. 'dd', 'MM', 'YYYY', etc.). If it contains such units an error is returned.
+
+[NOTE]
+Format specifier `F` will be working similar to format specifier `f`.
+It represents the most significant digit of the seconds fraction; that is,
```

Comment:
`... seconds fraction and all trailing `0`s will be printed to meet the number of `F`s in the pattern, e.g.:
for a time part `10:20:30.1234` and pattern `HH:mm:ss.FFFFFF`, the output string of the function would be: 
`10:20:30.123400`.

Reply:
Corrected note